content/en/docs/concepts/services-networking/dual-stack.md

```diff
@@ -65,12 +65,11 @@ To configure IPv4/IPv6 dual-stack, set dual-stack cluster network assignments:
 * kube-proxy:
   * `--cluster-cidr=<IPv4 CIDR>,<IPv6 CIDR>`
 * kubelet:
-  * when there is no `--cloud-provider` the administrator can pass a comma-separated pair of IP
-    addresses via `--node-ip` to manually configure dual-stack `.status.addresses` for that Node.
-    If a Pod runs on that node in HostNetwork mode, the Pod reports these IP addresses in its
-    `.status.podIPs` field.
-    All `podIPs` in a node match the IP family preference defined by the `.status.addresses`
-    field for that Node.
+  * `--node-ip=<IPv4 IP>,<IPv6 IP>`
+    * This is required for "bare metal" dual-stack nodes (i.e., nodes with no
+      `--cloud-provider`). If you are using a cloud provider, you only need to pass an
+      explicit `--node-ip` if you want to override the node IPs chosen by the cloud provider.
+    * (The legacy built-in cloud providers do not support dual-stack `--node-ip`.)
```

Comment:
Not sure about wording here... but it seemed like it made more sense to describe the non-legacy-cloud-provider behavior first and then add a disclaimer about the legacy cloud providers, so we can just remove that sentence once they go away?